packages/shared/src/models/examples.ts

```diff
@@ -0,0 +1,59 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export interface Category {
+  id: string;
+  name: string;
+  description?: string;
+}
+
+// validator for certain types arm / amd64
+export type ArchitectureType = 'arm64' | 'amd64';
+
+export interface Example {
+  id: string;
+  name: string;
+  categories: string[];
+  description: string;
+  repository: string;
```

Comment:
The first few are obvious, but maybe better to have a comment on repo to be overly clear if it is image or git repo.

Reply:
Done!